recipes/cesium/meta.yaml

```
@@ -0,0 +1,52 @@
+{% set name = "cesium" %}
+{% set version = "0.9.9" %}
+
+package:
+  name: "{{ name|lower }}"
+  version: "{{ version }}"
+
+source:
+  url: "https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz"
+  sha256: "7cbde6817c00e1f0c3e51aaf860cf7784dfc58766f2d4e68a6ca613be10d3d81"
+
+build:
+  number: 0
```

You can use the following for skipping Windows:
```suggestion
  number: 0
  skip: True  # [win]
```

Or if you only want to skip VC 2008:

```suggestion
  number: 0
  skip: True  # [win and vc<14]
```